tests/integration/suite/daprd/scheduler/streaming/stream.go

```diff
@@ -0,0 +1,162 @@
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package streaming
+
+import (
+	"context"
+	"fmt"
+	"runtime"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"google.golang.org/protobuf/types/known/anypb"
+
+	runtimev1pb "github.com/dapr/dapr/pkg/proto/runtime/v1"
+	"github.com/dapr/dapr/tests/integration/framework"
+	"github.com/dapr/dapr/tests/integration/framework/process/daprd"
+	"github.com/dapr/dapr/tests/integration/framework/process/exec"
+	"github.com/dapr/dapr/tests/integration/framework/process/logline"
+	"github.com/dapr/dapr/tests/integration/framework/process/ports"
+	"github.com/dapr/dapr/tests/integration/framework/process/scheduler"
+	"github.com/dapr/dapr/tests/integration/suite"
+)
+
+func init() {
+	suite.Register(new(streaming))
+}
+
+type streaming struct {
+	daprdA     *daprd.Daprd
+	daprdB     *daprd.Daprd
+	schedulers []*scheduler.Scheduler
+
+	streamloglineDaprA *logline.LogLine
+	streamloglineDaprB *logline.LogLine
+}
+
+func (s *streaming) Setup(t *testing.T) []framework.Option {
+	if runtime.GOOS == "windows" {
+		t.Skip("Skipping test on windows which relies on unix process signals")
+	}
+
+	s.streamloglineDaprA = logline.New(t,
+		logline.WithStdoutLineContains(
+			`Received response: [type.googleapis.com/google.type.Expr]:{} map[appID:A namespace:A]`,
+		),
+	)
+
+	s.streamloglineDaprB = logline.New(t,
+		logline.WithStdoutLineContains(
+			`Received response: [type.googleapis.com/google.type.Expr]:{} map[appID:B namespace:B]`,
+		),
+	)
+
+	fp := ports.Reserve(t, 6)
+	port1, port2, port3 := fp.Port(t), fp.Port(t), fp.Port(t)
+
+	opts := []scheduler.Option{
+		scheduler.WithInitialCluster(fmt.Sprintf("scheduler0=http://localhost:%d,scheduler1=http://localhost:%d,scheduler2=http://localhost:%d", port1, port2, port3)),
+		scheduler.WithInitialClusterPorts(port1, port2, port3),
+	}
+
+	clientPorts := []string{
+		"scheduler0=" + strconv.Itoa(fp.Port(t)),
+		"scheduler1=" + strconv.Itoa(fp.Port(t)),
+		"scheduler2=" + strconv.Itoa(fp.Port(t)),
+	}
+	s.schedulers = []*scheduler.Scheduler{
+		scheduler.New(t, append(opts, scheduler.WithID("scheduler0"), scheduler.WithEtcdClientPorts(clientPorts))...),
+		scheduler.New(t, append(opts, scheduler.WithID("scheduler1"), scheduler.WithEtcdClientPorts(clientPorts))...),
+		scheduler.New(t, append(opts, scheduler.WithID("scheduler2"), scheduler.WithEtcdClientPorts(clientPorts))...),
+	}
+
+	schedulerAddresses := []string{s.schedulers[0].Address(), s.schedulers[1].Address(), s.schedulers[2].Address()}
+
+	s.daprdA = daprd.New(t,
+		daprd.WithAppID("A"),
+		daprd.WithNamespace("A"),
```

Comment:
its necessary for now since I am ensuring that the logline contains the proper appID for each sidecar. It can be removed once the sidecar sends the job back to the proper app and there is a test for that, but for now the only way I can ensure via test that the proper sidecar is streamed back to is by the logline with the appID + ns, but I cant give the logline the appID + namespace dynamically before its created, so I cant ref `s.daprdA.AppID()` and add it to the logline before I create daprd. 

I will add a todo to update this once streaming to the app is there and tested.